### PR TITLE
Users frontend scenario- gherkin

### PR DESCRIPTION
**Pull request by pritalpatel**
Pull Request for Issue #16.
#### Summary of Changes

Create an users_frontend.feature file.

### CODE REVIEW

File: tests/acceptance/users_frontend.feature [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+     And I change name as a "patidar"
+     When I press the "submit"
+     Then Go to the user manager page in administrator
+     And I search the user with name "patidar"
+     And I sholud see the name "patidar"
+
+  Scenario: Test last login date
+    Given Needs to user "patel" logged in at least once
+    When I login as a super admin from backend
+    And Go to user manager page in administrator
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature]: You already said that you are super admin in the administrator area (Backend) :-)

No need to say it again -> And I go to the user manager page

File: tests/acceptance/users_frontend.feature [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    Then I search for "patel" username
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature]: And I search for .. -> This is not the assertion

File: tests/acceptance/users_frontend.feature [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    And I should see his last login date
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature]: Then i should see the last login date

The assertion (what we want to test for)

File: tests/acceptance/users_frontend.feature [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+  Background:
+    Given Joomla CMS is installed
+    Then I see the joomla! Home page
+
+  Scenario: Create user from frontend (index.php?option=com_users)
+    Given I click on the link "Create an account"
+    And I create a user with fields Name "patel", Username "patel", Password "patel" and Email "patel@gmail.com"
+    When I press the "Register"
+    Then I see the warning and notice message
+    But user is created
```
Comment (javigomez) [inline tests/acceptance/users_frontend.feature:15]: Wouldn't that be `And` instead of `But`?

File: tests/acceptance/users_frontend.feature [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+    And I active the user "patel"
+    When A login user "patel" with password "patel"
+    Then He should see the message "Hi patel,"
+
+   Scenario: Change user details in the frontend, check in the backend
+    Given I logged with user "patel"
+     And I press the "Edit Profile"
+     And I change name as a "patidar"
+     When I press the "submit"
+     Then Go to the user manager page in administrator
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature:39]: This is still when (And in your case) and you need to switch to an admin account

File: tests/acceptance/users_frontend.feature [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+     And I search the user with name "patidar"
+     And I sholud see the name "patidar"
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature:41]: This is your Then

File: tests/acceptance/users_frontend.feature [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+
+  Scenario: Login with created user to assure it is blocked
+    Given A newly created user "patel" with password "patel"
+    When He press the "login"
+    Then He should see the "Login denied! Your account has either been blocked or you have not activated it yet." warning
+
+  Scenario: Check if block and activation are working
+    Given There is a user manager page in administrator
+    And I unblock the user "patel"
+    And I active the user "patel"
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature:30]: active -> activate

File: tests/acceptance/users_frontend.feature [hunk 8]
```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given Joomla CMS is installed
```
Comment (yvesh) [inline tests/acceptance/users_frontend.feature:7]: This can be deleted

File: tests/acceptance/users_frontend.feature (hunk 6 above)
Comment (javigomez) [inline tests/acceptance/users_frontend.feature:41]: note that there is a typo `sholud` instead of `should`

**Comment by pritalpatel** [comment]
Please consider #51 for valid PR instead of this as it was having already merged commits. Closing this.

**Pull request closed without merging**